lib/streamlit/elements/data_frame_proto.py

```diff
@@ -131,9 +141,6 @@ def marshall_data_frame(data, proto_df):
     # Convert df into an iterable of columns (each of type Series).
     df_data = (df.iloc[:, col] for col in range(len(df.columns)))
 
-    import numpy as np
-    import pandas as pd
-
```

Comment:
These imports were unused

Reply:
Glad to remove them!

Tangential: I've often seen unused or "misordered" imports lying throughout or codebase; wonder if there's a way for our static analysis tool to find these for us. Tracking ticket filed https://github.com/streamlit/streamlit/issues/1808 a long time ago, dunno when we'll get to it though...

Reply:
PyCharm catches these (and automatically cleans them up, if you ask it to). That (probably) doesn't help us with auto-analysis on CircleCI, but it is fantastically useful while coding.